dockerfile

```diff
@@ -6,6 +6,7 @@ FROM --platform=linux/amd64 ubuntu:22.04 AS napari
 # below env var required to install libglib2.0-0 non-interactively
 ENV TZ=America/Los_Angeles
 ARG DEBIAN_FRONTEND=noninteractive
+ARG NAPARI_COMMIT=main
```

Comment:
This provides only default value?

Reply:
Yep, if not provided via CLI, this (`main`) is the default value.